run_llm/benchmark_set_up.py

```diff
@@ -0,0 +1,54 @@
+import json
+import numpy as np
+
+from pathlib import Path
+from abc import ABC, abstractmethod
+from typing import Optional
+
+
+class Benchmark(ABC):
+    data: dict
+    prompt: str
+    labels: Optional[str] = None
+    meshes: Optional[str] = None
```

Comment:
`labels: str | None = None` and `meshes: str | None = None` is preferred (PEP 604).

Reply:
Yes, I'll fix it. Just thought that Optional only works for version >= 3.10, so that it could cause issues for some users.